### DISCUSSION TITLE
Amazon Dataset

### DISCUSSION THREAD

**Discussion by prejieunt** [pyg-team/pytorch_geometric#8129] [Q&A]
Hi~ Thank for providing the code.
I have questions about Amazon Book Dataset. What is the train.txt file? Does that mean a test dataset in machine learning? If so, I guess both Amazon train dataset and test datasets are actually customer-evaluated data.

It is difficult for me to understand the process by which theories are coded, even though I have referred to related papers. In particular, it is difficult to understand the codes in which data become nodes. I'd like to get recommendations for reference materials.

**Comment by rusty1s** [comment]
The `AmazonBook` dataset splits edges into training edges and test edges, which are given by `data.edge_index` (the edges we are allowed to perform message passing on), and `data.edge_label_index` (the new/unseen edges we are trying to predict). I hope this answers your question :)
